creatures.css

```diff
@@ -0,0 +1,59 @@
+* {
+    box-sizing: border-box;
+  }
+
+header {
+    width: 100%;
+    height: 250px;
+    background-image: url("header.gif");
+    background-repeat: no-repeat;
+    background-size: 1800px;
+    margin-bottom: 40px;
+    border-radius: 10px;
+}
+
+form {
+    padding-top: 20px;
+    text-align: center;
+    font-family: AppleGothic;
```

Comment:
If AppleGothic is the only one being used, or mostly the only one, could put it in body instead of repeating it